src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/PitestPluginGeneralFunctionalSpec.groovy

```diff
@@ -112,6 +112,24 @@ class PitestPluginGeneralFunctionalSpec extends AbstractPitestFunctionalSpec {
             htmlFileWithReportForHelloPit.text.contains("System.out.println(&#34;Mutation to survive&#34;);")
     }
 
+    @Issue("https://github.com/szpak/gradle-pitest-plugin/issues/67")
+    void "reuses cached output"() {
+        given:
+        buildFile << getBasicGradlePitestConfig()
+        and:
+        writeHelloPitClass()
+        writeHelloPitTest()
+        when:
+        ExecutionResult result = runTasksSuccessfully('pitest', '--build-cache')
+        ExecutionResult result2 = runTasksSuccessfully('clean', 'pitest', '--build-cache')
+        then:
+        result.wasExecuted(':pitest')
+        result.getStandardOutput().contains("Stored cache entry for task ':pitest'")
+        and:    //it works with @OutputFile by default, but just in case
+        result2.wasExecuted(':pitest')
+        result2.getStandardOutput().contains("Task :pitest FROM-CACHE")
```

Comment:
Nice. Just please reformat that test to be consistent with the others.

In Idea:
https://solidsoft.wordpress.com/2014/05/20/more-compact-automatic-spock-specification-formatting-in-idea-13/

Reply:
Thanks for the IDEA advice, I've updated it. I'm just very used to having automatic code style formatting and/or verification when I run `build` :)